src/components/error-page/ErrorPage.tsx

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import { useStyletron } from "baseui";
+import { styles as s } from "./styles";
+import Block from "./ui/Block";
+import { HeadingXXLarge, ParagraphMedium, ParagraphSmall } from "baseui/typography";
+import { getErrorCodeOverrides, getErrorDescriptionOverrides, getRedirectOverrides } from "./overrides";
+import { ArrowUpIcon } from "../icons";
+import WhiteRect from "./ui/WhiteRect";
+
+export type ErrorPageProps<T extends React.ElementType = React.ElementType> = {
+  errorCode?: number;
+  errorDescription?: string;
+  redirectPath?: string;
```

Comment:
It's kinda opinion what it could be here. Because if it's something wrong with database we surely need to have here a refresh button not a redirect one. Redirect is only applied to 404 page. So if we want to use it as general thing here should be something like actionElement here or children that will be used here. And specific view of redirect should be moved to sub component like ErrorPage.Redirect element here.

Reply:
Added `actionComponent` prop. When `actionComponent` redirectPath and redirectTitle are restricted. If not - default `Redirect` component will be rendered.